ci_framework/roles/hive/tasks/add_bridges.yml

```diff
@@ -0,0 +1,45 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Setup the external bridge
+  community.general.nmcli:
```

Comment:
I think you need `become: true` there.

Reply:
Initially, I had `become` at the top level of the play. Hence, didn't have it in the lower levels. 

I believe that was not the right approach. Now, individual tasks have this statement.